ecmascript/transforms/src/optimization/simplify/dce/mod.rs

```diff
@@ -804,7 +985,7 @@ impl Dce<'_> {
     //            dropped: false,
     //        };
     //
-    //        let ret = op(&mut child);
+    //        let ret = op(&mut  child);
```

Comment:
```suggestion
    //        let ret = op(&mut child);
```
why use double space?

Reply:
963line and 975line too. 😆